runtime-common/stdlib/msgpack/object_visitor.h

```diff
@@ -65,10 +67,19 @@ class object_visitor : vk::not_copyable {
   void parse_error(size_t /*parsed_offset*/, size_t /*error_offset*/) const;
   void insufficient_bytes(size_t /*parsed_offset*/, size_t /*error_offset*/) const;
 
+  bool has_error() const {
```

Comment:
Can we not add this method? It seems enough to just check if an optional has value in place

Reply:
done